packages/appsync-emulator-serverless/bin/server.js

```diff
@@ -63,15 +63,16 @@ const main = async () => {
   const serverless = path.join(path.dirname(pkgPath), 'serverless.yml');
   const server = await createServer({ serverless, port, dynamodb });
   console.log('started at url:', server.url);
-  dynamodb_port && console.log(
-    `dynamodb config:
+  if (dynamodbPort)
```

Comment:
add `/* eslint-disable no-console */` before line 65 and it should build. Sorry for all the tedium!

Reply:
no problem - all my fault ;-)